functions/src/index.ts

```diff
@@ -21,7 +21,33 @@ main.use(bodyParser.json());
 export const getCVEFunction = CVEFunction.getCVE;
 export const getSPLFunction = SPLFunction.getSPL;
 export const getBulletinFunction = bulletinFunction.getBulletin;
-export const getAndroidVersionFunction = androidVersionFunction.getAndroidVersion;
-export const storeEmailFunction = notificationFunction.accountCreate;
+export const getAndroidVersionFunction =
+  androidVersionFunction.getAndroidVersion;export const storeEmailFunction = notificationFunction.accountCreate;
 export const notifyNewVersionFunction = notificationFunction.notifyNewVersion;
 export const notifyNewReleaseFunction = notificationFunction.notifyNewRelease;
+
+export const getData = functions.https.onRequest(main);
+
+app.post('/data', (request: any, response: any) => {
```

Comment:
Give it a more meaningful path, like /grantadminrole

Also, since this function is essentially a security hole, add a comment
"DO NOT DEPLOY TO PRODUCTION" or something similar.